src/c/config.c

```diff
@@ -597,11 +614,13 @@ void edgex_device_updateConf (void *p, const devsdk_nvpairs *config)
 {
   iot_data_map_iter_t iter;
   bool updatedriver = false;
+  bool updatemetrics = false;
   devsdk_service_t *svc = (devsdk_service_t *)p;
 
   iot_log_info (svc->logger, "Reconfiguring");
 
   edgex_device_overrideConfig_nvpairs (svc->config.sdkconf, config);
+  updatemetrics = strcmp (svc->config.metrics.interval, iot_data_string_map_get_string (svc->config.sdkconf, DYN_PREFIX "Telemetry/Interval"));
```

Comment:
I thought only Writable/xxx or Driver/xxx elements get added to the map of parameters in config?

Reply:
DYN_PREFIX is "Writable/"